auth-api/src/auth_api/resources/entity.py

```diff
@@ -80,10 +80,10 @@ def get(business_identifier):
         return response, status
 
     @staticmethod
-    @_JWT.requires_auth
-    @_JWT.has_one_of_roles([Role.SYSTEM.value])
     @TRACER.trace()
-    def put(business_identifier):
+    @cors.crossdomain(origin='*')
+    @_JWT.requires_auth
+    def patch(business_identifier):
```

Comment:
What i thought was to keep the put and write another patch... may be check with @sumesh-aot 
either way is good..

Reply:
No other code use PUT endpoint. So, I switch the endpoint to PATCH.